### PR TITLE
kramo.Cartridges version v1.3.2

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/101877)

### CODE REVIEW

File: manifests/k/kramo/Cartridges/v1.3.2/kramo.Cartridges.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: kramo.Cartridges
+PackageVersion: v1.3.2
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/kra-mo/cartridges/releases/download/v1.3.2/Cartridges.Setup.exe
+  InstallerSha256: 5D7FED7EE8BA32D7661E1602EE0F0EBD551A0911C25EA9987A8154CF57F5C689
```
Comment (stephengillie) [inline manifests/k/kramo/Cartridges/v1.3.2/kramo.Cartridges.installer.yaml:10]:
```suggestion
InstallerType: inno
Installers:
- Architecture: x86
  Scope: user
  InstallerUrl: https://github.com/kra-mo/cartridges/releases/download/v1.3.2/Cartridges.Setup.exe
  InstallerSha256: 5D7FED7EE8BA32D7661E1602EE0F0EBD551A0911C25EA9987A8154CF57F5C689
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x86
  Scope: machine
  InstallerUrl: https://github.com/kra-mo/cartridges/releases/download/v1.3.2/Cartridges.Setup.exe
  InstallerSha256: 5D7FED7EE8BA32D7661E1602EE0F0EBD551A0911C25EA9987A8154CF57F5C689
  InstallerSwitches:
    Custom: /ALLUSERS
```

Comment (stephengillie) [reply manifests/k/kramo/Cartridges/v1.3.2/kramo.Cartridges.installer.yaml:10]: Currently the package prompts for scope: 
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230407185551
``

Comment (kbdharun) [reply manifests/k/kramo/Cartridges/v1.3.2/kramo.Cartridges.installer.yaml:10]: Thanks for the help, I will apply the changes.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
This package is expecting arguments:
```
Usage:
  gdbus.exe COMMAND

Commands:
  help         Shows this information
  introspect   Introspect a remote object
  monitor      Monitor a remote object
  call         Invoke a method on a remote object
  emit         Emit a signal
  wait         Wait for a bus name to appear
...
```

**Pull request merged**